modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/ProductionDBSync_conf.pm

```diff
@@ -38,6 +43,7 @@ sub default_options {
     populate_controlled_tables    => 1,
     populate_analysis_description => 1,
     group => [],
+    email_on_completion => 0,
```

Comment:
I would personally turn it on by default.

Reply:
In general, our pipelines don't email you when they complete - it's okay for us, because we tend to do a small number of pipelines on divisions. But other teams might be running this per-species, and lots of confirmation emails might be annoying, so I prefer to keep it as opt-in.